### PR TITLE
feat: Location Entity

### PR DESCRIPTION
**Pull request by cdnninja**
Still requires a bit of API work for this value to exist in the car.    Will post for review purposes. Won't merge until I have the API working on this as well.

### CODE REVIEW

**Comment by cdnninja** [comment]
Should icon, and source type move to attributes since they don't change?

File: custom_components/kia_uvo/device_tracker.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    @property
+    def latitude(self):
+        return self.vehicle.location_latitude
+
+    @property
+    def longitude(self):
+        return self.vehicle.location_longitude
+
+    @property
+    def source_type(self):
```
Comment (fuatakgun) [inline custom_components/kia_uvo/device_tracker.py:57]: Nit: let's use attr field

File: custom_components/kia_uvo/device_tracker.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    entities = []
+    for vehicle_id in coordinator.vehicle_manager.vehicles.keys():
+        vehicle: Vehicle = coordinator.vehicle_manager.vehicles[vehicle_id]
+        entities.append(HyundaiKiaConnectLocationTracker(coordinator, vehicle))
+
+    async_add_entities(entities)
+    return True
+
+
+class HyundaiKiaConnectLocationTracker(TrackerEntity, HyundaiKiaConnectEntity):
```
Comment (fuatakgun) [inline custom_components/kia_uvo/device_tracker.py:36]: nit: HyundaiKiaConnectTracker or HyundaiKiaConnectDeviceTracker

**Review by fuatakgun — CHANGES_REQUESTED** [review]
Perfect

File: custom_components/kia_uvo/device_tracker.py (hunk 1 above)
Comment (cdnninja) [reply custom_components/kia_uvo/device_tracker.py:57]: https://developers.home-assistant.io/docs/core/entity/device-tracker/ states the following so I don't think I can. 

`CAUTION
TrackerEntity does not support attribute shorthand for property implementation`

**Pull request merged**